<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquationsClass</class>
 <widget class="QMainWindow" name="EquationsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equations</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidgetEq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSol">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute solution</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpdate">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update equation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coef1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCoef1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coef2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCoef2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coef3:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCoef3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUpdated">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Equations.qrc"/>
 </resources>
 <connections/>
</ui>
